integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/AbstractCloudProviderAssertion.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.it.cloudbreak.cloud.v4;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.it.cloudbreak.dto.environment.EnvironmentTestDto;
+
+@Component
```

Comment:
Component is unnecessary and misconfigured, abstract class cannot be initiated by spring context